data/tlg4027/tlg001/tlg4027.tlg001.opp-grc1.xml

```diff
@@ -49,43 +50,39 @@
             </titleStmt>
             <publicationStmt>
                 <authority>University of Leipzig</authority>
-                <idno type="filename">commentaria_23.xml</idno>
+                <idno type="filename">tlg4027.tlg001.opp-grc1.xml</idno>
                 <availability>
                     <p>
                         <ref target="https://creativecommons.org/licenses/by-sa/4.0/">Available
                             under a Creative Commons Attribution-ShareAlike 4.0 International
                             License</ref></p>
                 </availability>
-                <date>2014</date>
+                <date>2016</date>
                 <publisher>University of Leipzig</publisher>
                 <pubPlace>Germany</pubPlace>
             </publicationStmt>
 
             <sourceDesc>
-                <listBibl xml:lang="lat">
+                <listBibl>
                     <biblStruct>
                         <monogr>
                             <editor>
                                 <persName>
                                     <name xml:lang="lat">Michael Hayduck</name>
-                                    <addName xml:lang="de">Michael Hayduck</addName>
                                 </persName>
                             </editor>
-                            <editor>
-                                <persName>
-                                    <name xml:lang="lat">Maximilianus Wallies</name>
-                                    <addName xml:lang="de">Max Wallies</addName>
-                                </persName>
-                            </editor>
-                            <author>Anonymus</author>
-                            <title>Categoriarum paraphrasis</title>
+                          <title xml:lang="lat">Paraphrasis categoriarum</title>
+                           <author xml:lang="lat">Anonymi in Aristotelis Categorias</author>
```

Comment:
epiDoc throws an error that `<author>` cannot be here. On my local copy, when I move the `<author>` element to before the `<title>` element, then it passes. That seems...odd.